tests/virt/node/descheduler/conftest.py

```
@@ -137,6 +141,17 @@ def node_with_max_memory_labeled_for_descheduler_test(node_with_most_available_m
         yield
 
 
+@pytest.fixture(scope="class")
+def unloaded_migration_target_node(admin_client, node_with_most_available_memory):
+    """Drain and uncordon node to reduce loadage."""
+    LOGGER.info(f"Draining node {node_with_most_available_memory.name} to reduce loadage")
+
+    with node_mgmt_console(admin_client=admin_client, node=node_with_most_available_memory, node_mgmt="drain"):
+        wait_for_node_schedulable_status(node=node_with_most_available_memory, status=False)
+
+    yield node_with_most_available_memory
```

Ruff (0.15.12)

[warning] No teardown in fixture `unloaded_migration_target_node`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)